docs/pages/linking/overview.mdx

```diff
@@ -101,6 +64,29 @@ This link type allows your app to open when a user clicks an HTTP(S) link pointi
   Icon={AppleIcon}
 />
 
+## Linking to your app from other apps or websites
+
+[Deep Links](https://en.wikipedia.org/wiki/Deep_linking) are links to a specific URL-based content inside an app or a website.
+
+For example, by clicking a product advertisement, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like (or alternatively be invoked by JavaScript with setting `window.location.href`):
+
+```html
+<a href="myapp://web-app.com/product">View Product</a>
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links. We recommend [universal linking](#universal-linking) for http(s)-based deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
 ## Use Expo Router to handle deep linking
```

Comment:
Can we move this section at the last? this is more like an additional information. I also wonder if this section should be part of the Deep linking section.

Reply:
I think it makes sense to have it here in this introduction; the reason it's here is that it's related to incoming links, and the new paragraph is about outgoing links, so putting it behind it would be weird imho

Reply:
In that case, let's make it a sub section for the for incoming links.